connect_proto.gemspec

```
@@ -5,7 +5,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |s|
   s.name        = "primary_connect_proto"
-  s.version     = "0.10.1"
+  s.version     = "0.10.2"
```

Nitpick: this is more than a bug fix, it's actually adding functionality, so I think it should be a minor version change and not a patch version

From [Semver docs](https://semver.org/) (emphasis added by me)

> Given a version number MAJOR.MINOR.PATCH, increment the:
> 
> -  MAJOR version when you make incompatible API changes,
> -  **MINOR version when you add functionality in a backwards compatible manner**, and
> -  PATCH version when you make backwards compatible bug fixes.


```suggestion
  s.version     = "0.11.0"
```

---

I'll change that, thanks!